apps/web/src/views/universalFarms/components/PositionItem.tsx

```diff
@@ -272,38 +368,28 @@ export const PositionItemDetail = ({
         secondaryToken={currencyBase.wrapped}
       />
       <DetailsContainer>
-        <Row gap="8px">
-          <Text bold>{`${currencyQuote.symbol} / ${currencyBase.symbol} LP`}</Text>
-          {tokenId ? <Text color="textSubtle">(#{tokenId.toString()})</Text> : null}
-          <FeeTier type={protocol} fee={fee} />
-          <TagCell>
-            {isStaked && (
-              <Tag variant="primary60" mr="8px">
-                {t('Farming')}
-              </Tag>
-            )}
-            {protocol === Protocol.V3 && <RangeTag lowContrast removed={removed} outOfRange={outOfRange} />}
-          </TagCell>
-        </Row>
-        <Row>{desc}</Row>
-        <Row gap="sm">
-          <FiatNumberDisplay
-            prefix="~"
-            value={totalPriceUSD}
-            style={{ color: theme.colors.textSubtle }}
-            showFullDigitsTooltip={false}
-          />
-          ({amount0 ? amount0.toFixed(6) : 0} {currencyQuote?.symbol ?? '-'} / {amount1 ? amount1.toFixed(6) : 0}{' '}
-          {currencyBase?.symbol ?? '-'})
-        </Row>
-
-        {pool ? (
-          <Row>
-            <Column style={{ fontWeight: 600 }}>APR: </Column>
-            <PoolApyButton pool={pool} />
-          </Row>
-        ) : null}
+        <Column>{detailInfo}</Column>
+        <Column justifyContent="flex-end">
+          {!isStaked && !removed ? (
+            <Button scale="md" width={['100px']} style={{ alignSelf: 'center' }} onClick={handleStakeAndCheckInactive}>
```

Comment:
Why there is a fixed width?

Reply:
I'll adapt the mobile UI later, so this can be left for now.